pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Managing schemas using Schema Registry
+description: The guide explains how you can manage schemas using Schema Registry.
+weight: 50
+---
+
+## Introduction
+You will learn how to manage schemas with Schema Registry. Managing schemas will enable us to ensure a consistent data exchange across distributed systems, thus minimizing errors.
+
+## Background context
+The need for schema management has become increasingly prevalent to handle the evolving complexity of modern Event-Driven Architecture. A Schema Registry is a centralized service that stores and maintains the schemas for data exchanged between various components of a modern distributed system. Validating data being exchanged helps in maintaining data consistency and compatibility. 
+
+While several Schema Registry implementations exist, you will use the Apicurio Registry in this tutorial. Apicurio Registry is a popular open-source Schema Registry implementation that supports multiple serialization formats and facilitates schema management for diverse data in distributed systems. You will be using the Apicurio Registry in combination with Avro, which is a language-neutral data serialization system.
+
+## Installation guide
+1. Install [Docker](https://docs.docker.com/engine/install/) from the official website.
+2. Open the command prompt and pull up the Apicurio Registry Docker Image from Docker Hub using the following command:
+    <CodeBlock language="bash">
+    {`docker pull apicurio/apicurio-registry`}
```

Comment:
you do not need to have separate pull instruction

`docker run` works in a way that it pulls by default if you do not have given image on local

why is what I meant in offline discussion that  to start registry, is just one command

Reply:
@derberg I have deleted that step now